cpplib/catch2utils/XStringFormat.inl

```diff
@@ -0,0 +1,32 @@
+template <typename T>
```

Comment:
A short banner of the file purpose would be nice… along with a URL to the string maker (or how is it called) documentation page.